Source/WebKit/GPUProcess/graphics/RemoteGraphicsContextGL.h

```diff
@@ -44,8 +44,10 @@
 #include <WebCore/GraphicsContextGLCocoa.h>
 #elif USE(LIBGBM)
 #include <WebCore/GraphicsContextGLGBM.h>
-#else
+#elif USE(ANGLE)
 #include <WebCore/GraphicsContextGLTextureMapperANGLE.h>
+#else
+#include <WebCore/GraphicsContextGLTextureMapper.h>
```

Comment:
https://github.com/WebKit/WebKit/pull/4374 is going to remove `GraphicsContextGLTextureMapper`.

Reply:
Ah ok I'm fine with dropping this if Igalia is removing the `!USE(ANGLE)` path for WebGL

Reply:
It can be dropped, it wouldn't ever work in GPUProcess anyway.